### DISCUSSION TITLE
Copilot chat not seen on vs 2022

### DISCUSSION THREAD

**Discussion by ciznerguy** [community/community#102745] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Not seeing any option to open copilot chat.
The only thing that works is using // to generate code.
nothing else.
To clarify copilot suggests code.
But no option for either Inline nor Chat help

**Comment by SwaminathanSR** [comment]
Face the same problem with VS 17.7.4. GitHub Copilot Chat is not listed even under "Manage Extensions".

I tried installing the extension directly from VS Marketplace web. The installation failed with the installation log saying 
"VSIXInstaller.NoApplicableSKUsException". 

My account does have CoPilot Chat enabled.

**Reply by ayim** [reply]
GH Copilot Chat requires v17.8 of Visual Studio, once you update, the extension will install.

**Comment by SwaminathanSR** [comment]
Upgrading the VS to 17.8.6 (latest version) got rid of the problem. Post VS upgrade, I was able to install and use the copilot chat.

**Reply by ciznerguy** [reply]
just finished upgrading to 17.8.6
still no chat

**Reply by SwaminathanSR** [reply]
Are you able to install the extension? if not, any clue from the install log file?
Once I got it installed successfully, I see it under the "View" menu.

**Comment by ciznerguy** [comment]
the extension is installed and even active.
I get great code suggestions 
no inline copilot nor chat very frustrated. Nothing on view menu

GitHub Copilot 1.155.0.0 (v1.155.0.0@1564940eb)
GitHub Copilot Agent 1.155.0 (1.155.0)
[DEBUG] [agent] [2024-02-02T18:41:01.418Z] Agent service starting
[DEBUG] [agent] [2024-02-02T18:41:01.740Z] Telemetry initialized
Auth Status: OK
Successfully authorized GitHub Copilot.

**Reply by ayim** [reply, marked as answer]
Did you try to reinstall the Chat extension after updating again?
It would not have installed on pre 17.8 version of VS.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat

**Reply by ciznerguy** [reply]
solved the problem. thanks

**Reply by CarlGraff** [reply]
How did yo solve it?
